arches/app/media/js/utils/map-popup-provider.js

```diff
@@ -40,6 +42,34 @@ define(['arches',
             return features;
         },
 
+        /**
+         * This method enables custom logic for how the feature in the popup should be handled and/or mutated en route to the mapFilter.
+         * @param popupFeatureObject - the javascript object of the feature and its associated contexts (e.g. mapCard).
+         * @required @method mapCard.filterByFeatureGeom()
+         * @required send argument: @param feature - a geojson feature object
+         * @optional send argument: @param resourceid
+         */
+        sendFeatureToMapFilter: function(popupFeatureObject)
+        {
+            let feature = popupFeatureObject.geometries()[0].geom.features[0];
```

Comment:
There is an assumption here that there is only one geometry per node. If multiple geometries exist it always selects the first one. If we use the geometry directly, not trying to use a resourceId & nodeId, there should be a way to pull the geometry selected (in processData() I think it can be found using `features.activeFeature.feature.toJSON().geometry.coordinates`).

Reply:
Good catch! I've altered that function to now expect the index of the corresponding feature to instruct it which one to pick off of `popupFeatureObject.geometries()` -- I've also include it as a param for `showFilterByFeature` just in case further inspection on that particular feature is needed to determine whether it should be eligible as a map filter